src/modules/dashboard/components/CoinMachine/TokenSalesTable/TokenSalesTable.tsx

```diff
@@ -169,7 +168,6 @@ const TokenSalesTable = ({
   useEffect(() => {
     if (periodRemainingMS > 1000 && periodRemainingMS < periodLengthMS) {
       setTimeout(() => {
-        refetchSalePeriodsData(salePeriodQueryVariables);
```

Comment:
But this is the tokens sales table, we need to refetch the periods data each time a period passes. 

What's reason to remove this ?

Reply:
If we poll data every "period length" then why we also want to refetch? 🤔

Reply:
Let's say the period is 1 hour.

You load the component when the period has only 30 minutes until it ends.

Your poll interval will run every hour, but it will not run in 30 minutes, when the period ends, and you want to refresh the data.

That's why we also use `refetch`, but be able to manually get new data, in-between periods, if we loaded the component in the time between periods